modules_core/styles.js

```diff
@@ -0,0 +1,34 @@
+var h = require('../h')
+var compile = require('micro-css')
+var fs = require('fs')
+var Path = require('path')
+
+// TODO distribute these styles across all
+// the relevant modules, not as a core style.
+var coreStyle = fs.readFileSync(Path.join(__dirname, '../style.css'))
+
+module.exports = {
+  needs: {
+    mcss: 'map',
+    css: 'map'
```

Comment:
guess  I wonder whether we should add css as an option - I think if we're doing mcss, lets stick to that and be pushed around by the opinions it has. I wouldn't want to see people adding piecemeal css just _because it's quicker_